packages/flutter_tools/test/general.shard/build_system/targets/ios_test.dart

```diff
@@ -1165,11 +1167,12 @@ void main() {
         environment.defines[kSrcRoot] = projectPath;
         environment.defines[kTargetDeviceOSVersion] = '18.4.1';
 
+        final StringBuffer buffer = await capturedConsolePrint(() async {
+          await const DebugIosLLDBInit().build(environment);
+        });
         expect(
-          const DebugIosLLDBInit().build(environment),
-          throwsToolExit(
-            message: 'Debugging Flutter on new iOS versions requires an LLDB Init File.',
-          ),
+          buffer.toString(),
```

Comment:
Silly nit: could push the `.toString()` to `capturedConsolePrint`'s implementation. This might be nice if we end up using `capturedConsolePrint` in more locations in the future. I don't feel strongly, feel free to skip